apps/ledger-live-mobile/src/components/WalletTab/WalletTabHeader.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useContext } from "react";
+import { Animated } from "react-native";
+import Header from "../../screens/Portfolio/Header";
+import { Box } from "../../../../../libs/ui/packages/native/lib";
+import { WalletTabNavigatorScrollContext } from "./WalletTabNavigatorScrollManager";
+
+function WalletTabHeader({ hidePortfolio }: { hidePortfolio: boolean }) {
+  const { scrollY, headerHeight } = useContext(WalletTabNavigatorScrollContext);
+  const y = scrollY.interpolate({
+    inputRange: [0, headerHeight],
+    outputRange: [0, -headerHeight],
+    extrapolateRight: "clamp",
+  });
+  return (
+    <Animated.View
+      style={[
+        {
+          top: 0,
+          height: headerHeight,
+          width: "100%",
+          position: "absolute",
+        },
+        { transform: [{ translateY: y }] },
+      ]}
+    >
```

Comment:
can't use `styled` like `Skelleton`?

Reply:
not sure it is useful, as i will not be used elsewhere, i will still need to inject style inside style prop for the animation, and this Header will never have style props ...